app/javascript/src/components/Navigation/components/NotificationItem.jsx

```diff
@@ -6,21 +6,71 @@ import styled from "styled-components";
 
 import CustomNavItem from "./CustomNavItem";
 import NotificationImage from "./NotificationImage";
+import {
+  MAIN_COLOR_BLUE,
+  COLOR_ORANGE,
+  ICON_GREEN_COLOR,
+  WILD_SAND_COLOR
+} from "../../../config/constants/colors";
 
 const StyledListItem = styled(CustomNavItem)`
   border-bottom: 1px solid #d2d2d2;
```

Comment:
if you imported a color you can use it here `1px solid ${WILD_SAND_COLOR}` or whatever color you need. I think that this styled components stuff also provide some themplating system so you can access colors with just importing a theme provider but I didn't use it yet. Maybe you can have a look at it if you want: **https://www.styled-components.com/docs/advanced#theming**